public/stylesheets/site/2.0/13-group-blurb.css

```diff
@@ -268,7 +268,7 @@ eg collections, users, skins, instead of the 4-icon list
 
 /*various little mods*/
 
-.collection .name {
+.heading {
```

Comment:
Since this is no longer specific to blurbs, it should probably be in this block in 02-elements.css: https://github.com/otwcode/otwarchive/blob/fa272977875c757b2cf899743e30beac5ae8a08e/public/stylesheets/site/2.0/02-elements.css#L53-L56

That way, it would also cover any heading tags used in work summaries. The _slight_ downside is it will also apply to heading tags used in works, and it's _possible_ that there are a few users with work skins who count on the headings not breaking, but I think that's such an edge case it's not worth coding an exception for.